diff --git a/dist/config-A-j1WmC5.mjs b/dist/config-A-j1WmC5.mjs
index f8bd0b61ef8c4c7a5aaab2592cf3147f8ed7cd1c..e311b6ee2fcdefb2c11bd07d5ee76251dc908dc1 100644
--- a/dist/config-A-j1WmC5.mjs
+++ b/dist/config-A-j1WmC5.mjs
@@ -6,7 +6,7 @@ import { bundle } from "@scalar/json-magic/bundle";
 import { fetchUrls, parseJson, parseYaml, readFiles } from "@scalar/json-magic/bundle/plugins/node";
 import { upgrade, validate } from "@scalar/openapi-parser";
 import { isNullish as isNullish$1, pick } from "remeda";
-import yaml from "js-yaml";
+import * as yaml from "js-yaml";
 import * as mock from "@orval/mock";
 import { dedupeStrictMockTypeDeclarations, generateFakerForSchemas, generateMockImports, getDefaultMockOptionsForType } from "@orval/mock";
 import angular from "@orval/angular";
